x-pack/tasks/helpers/globs.js

```diff
@@ -28,8 +28,7 @@ function getPluginPaths(plugins, opts = {}) {
 
     paths = paths.concat([indexPaths, commonPaths, serverPaths]);
     if (plugin === 'code') {
-      // TODO: Fix and reenable Code test
-      // paths.push(`legacy/plugins/${serverPath}/**/${testPath}/*.ts`);
+      paths.push(`legacy/plugins/${serverPath}/**/${testPath}/*.ts`);
```

Comment:
@zfy0701  the reason for the mocha test failures [here](https://github.com/elastic/kibana/pull/39721) is not caused by the path changes, but because the `env.dev` config was not correctly set up in other tests: https://github.com/elastic/kibana/blob/3a1842c04d91b248ae0d8d9f831c42c94ddad289/x-pack/legacy/plugins/code/server/lsp/language_servers.ts#L110. I am wondering why the change to bring this config can pass the CI. The [change](https://github.com/elastic/kibana/pull/39421) was made before the PR which disables code mocha test was submitted.